.luacheckrc

```diff
@@ -1,12 +1,13 @@
 
 read_globals = {
-	"vector",
-	"screwdriver",
-	"minetest",
-	"default",
-	"pipeworks",
-	"dump",
-	"VoxelArea",
+    "vector",
+    "screwdriver",
+    "minetest",
+    "default",
+    "pipeworks",
+    "dump",
+    "VoxelArea",
+    "ItemStack",
```

Comment:
Why not tabs?

Reply:
AFAIK, I'm using tabs here.

Reply:
Those are spaces, not tabs, I can select each space individually.

Reply:
Probably there's something messed up with my IDE...I am using TABS according to my IDE?

Reply:
Should be fixed now.